iotfunctions/pipeline.py

```diff
@@ -192,7 +192,7 @@ def execute(self, df=None, to_csv=False, dropna=False, start_ts = None, end_ts =
             try:
                 msg = ' Dataframe at start of %s: ' %s.__class__.__name__
                 last_msg = s.log_df_info(df,msg)
-            except AttributeError:
+            except Exception:
```

Comment:
This is kind of an antipattern to go from explicit to implicit... https://realpython.com/the-most-diabolical-python-antipattern/  What exception are you expecting?  Why not use __except (AttributeError, ExpectedError):__ instead?